### PR TITLE
Added badges for AppVeyor

### PR DESCRIPTION
**Pull request by morozov**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -23,16 +23,22 @@ Powerful database abstraction layer with many features for database schema intro
...
+  [AppVeyor master image]: https://ci.appveyor.com/api/projects/status/i88kitq8qpbm0vie/branch/master?svg=true
   [ContinuousPHP]: https://continuousphp.com/git-hub/doctrine/dbal
   [2.5 image]: https://img.shields.io/travis/doctrine/dbal/2.5.svg?style=flat-square
   [Coverage 2.5 image]: https://img.shields.io/scrutinizer/coverage/g/doctrine/dbal/2.5.svg?style=flat-square
   [Quality 2.5 image]: https://img.shields.io/scrutinizer/g/doctrine/dbal/2.5.svg?style=flat-square
   [ContinuousPHP 2.5 image]: https://img.shields.io/continuousphp/git-hub/doctrine/dbal/2.5.svg?style=flat-square
   [2.5]: https://github.com/doctrine/dbal/tree/2.5
   [Scrutinizer 2.5]: https://scrutinizer-ci.com/g/doctrine/dbal/?branch=2.5
+  [AppVeyor 2.5]: https://ci.appveyor.com/project/doctrine/dbal/branch/2.5
+  [AppVeyor 2.5 image]: https://ci.appveyor.com/api/projects/status/i88kitq8qpbm0vie/branch/2.5?svg=true
```
Comment (Ocramius) [inline README.md]: We don't test `2.5.x` on AppVeyor AFAIK

**Review by Ocramius — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (Majkl578) [reply README.md]: 2.6 either, only master (2.7) currently.

**Comment by morozov** [comment]
@Ocramius Y U NO want an extra badge in the README?

**Review by Ocramius — APPROVED** [review]

**Comment by Ocramius** [comment]
I need regilar reminders and electroshocks to merge stuff, @morozov 😂

**Pull request merged**